XPI/index.js

```diff
@@ -120,8 +120,8 @@ tabs.on('activate', function() {
 });
 
 function getActiveWorker() {
-	let tab = tabs.activeTab;
-	for (let i in workers) {
+	const tab = tabs.activeTab;
+	for (const i in workers) {
```

Comment:
Even though this is correct, Firefox (42) seems to only work if it's `let` ...dammit Firefox.
There are two other `for (const ...` that need to be changed too.